spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RootController.java

```diff
@@ -159,6 +159,7 @@ public RootResource info() {
 			root.add(linkTo(methodOn(TasksInfoController.class).getInfo(null, null, null)).withRel("tasks/info/executions"));
 			root.add(linkTo(methodOn(TaskLogsController.class).getLog(null, null, null)).withRel("tasks/logs"));
 			root.add(linkTo(methodOn(TaskExecutionThinController.class).listTasks(null, null)).withRel("tasks/thinexecutions"));
+			root.add(linkTo(methodOn(TaskExecutionThinController.class).retrieveTasksByName(null, null, null)).withRel("tasks/thinexecutions/name"));
 			if (featuresProperties.isSchedulesEnabled()) {
```

Comment:
Not related to this file, but rather `TaskExecutionController`... the port source has this added code https://github.com/spring-cloud/spring-cloud-dataflow/pull/5994/files#diff-2cc7b0286a71d935467884c16b01bf672821257fc6005c25df49141dbd8e7b7dR188-R191